build.gradle

```diff
@@ -169,8 +169,8 @@ tasks.register("verifyVersions") {
  * after the backport of the backcompat code is complete.
  */
 
-boolean bwc_tests_enabled = true
-String bwc_tests_disabled_issue = "" /* place a PR link here when committing bwc changes */
+boolean bwc_tests_enabled = false
+String bwc_tests_disabled_issue = "https://github.com/elastic/elasticsearch/pull/68760" /* place a PR link here when committing bwc changes */
```

Comment:
If you like you could switch to referencing https://github.com/elastic/elasticsearch/pull/68811 as that contains the fix.